scripts/codegen/create_fortran_kernel_interface.py

```diff
@@ -142,3 +145,40 @@
         # Write remainder of the input file
         for line in f_in:
             f.write(line)
+
+    # Generate interface functions with fixed arity to safely interface C with Fortran
+    fname = os.path.join(C_SRC_DIR, 'kernel_fortran_interface.cpp')
+    print('Write OCCA Fortran kernel interface C source to: %s' % fname)
+    with open(fname, 'w') as f:
+        f.write(generate_edit_warning(None, comment_marker='//'))
+        f.write('\n')
+        f.write('#include <occa/c/kernel_fortran_interface.h>\n')
+        f.write('#include <occa/c/kernel.h>\n')
+        f.write('\n')
+        f.write('OCCA_START_EXTERN_C\n')
+        f.write('\n')
```

Comment:
Nit: We can use the `'''` to avoid having to type `f.write` once per line
```suggestion
        f.write('''
#include <occa/c/kernel_fortran_interface.h>
#include <occa/c/kernel.h>

OCCA_START_EXTERN_C
        
''')
```